temp/legacy-code/src/main/java/com/ivy/legacy/data/DueSection.kt

```diff
@@ -5,9 +5,17 @@ import com.ivy.base.legacy.Transaction
 import com.ivy.wallet.domain.pure.data.IncomeExpensePair
 import kotlinx.collections.immutable.ImmutableList
 
+@Deprecated("Uses legacy Transaction")
 @Immutable
-data class DueSection(
+data class LegacyDueSection(
     val trns: ImmutableList<Transaction>,
     val expanded: Boolean,
     val stats: IncomeExpensePair
 )
+
+@Immutable
+data class DueSection(
```

Comment:
Please, keep the deprecation - ideally, we'd like to get rid of these UI layer models and replace them with more optimal ones

Reply:
Done

Reply:
@michalguspiel just fyi when you're ready for re-review click the ♻️ refresh button next to my regiew. Also feel free to resolve all fixed comments